app/controllers/with_user_controller.rb

```diff
@@ -7,6 +7,13 @@ def authorize!
     # requested is authorized if using okta to provide a valid user id
     return if authorization && current_user
 
+    # currently, if trying to operate on a specific @user, you can only operate on your own user data, but this may change later
+    return if @user && @user == current_user
```

Comment:
Your new UsersController does not `set_user`; will this behave properly there?

Reply:
Right, I realized that and did some more work on this, I'll re-request the review